tests/Fixtures/ConvertConfig/custom_extension.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+use Behat\Behat\Output\Printer\ConsoleOutputFactory;
+use Behat\Testwork\Output\Formatter;
+use Behat\Testwork\Output\Printer\OutputPrinter;
+use Behat\Testwork\Output\Printer\StreamOutputPrinter;
+use Behat\Testwork\ServiceContainer\Extension;
+use Behat\Testwork\Output\ServiceContainer\OutputExtension;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+class CustomExtension implements Extension
```

Comment:
This file is used in some tests. When we define an extension, we need to have a real extension that can be loaded. Also, if we define a custom formatter, this formatter needs to exist